cypress/e2e/tests/migration/applicationinventory/assessment/review/filter.test.ts

```diff
@@ -30,11 +32,13 @@ import { Stakeholders } from "../../../../../models/migration/controls/stakehold
 import { Application } from "../../../../../models/migration/applicationinventory/application";
 import { AssessmentQuestionnaire } from "../../../../../models/administration/assessment_questionnaire/assessment_questionnaire";
 import { identifiedRisksFilterValidations } from "../../../../../views/reportsTab.view";
+import { Archetype } from "../../../../../models/migration/archetypes/archetype";
 
 let application: Application;
 let stakeholder: Stakeholders;
+let archetype: Archetype;
 
-// Polarion TC 495
+// Polarion TC 495 and TC 541
 describe(["@tier2"], "Review Identified Risks filter validations", function () {
```

Comment:
Pls update this to -
'Assessment filter validation on Application and Archetype review page'

Reply:
Another minor change. The tests should be tier3. Otherwise the PR looks LGTM

Reply:
Title " Review Identified Risks filter validations for assessments" seems more appropriate .

Reply:
done!